lib/logitech_receiver/descriptors.py

```diff
@@ -213,7 +213,14 @@ def _D(name, codename=None, kind=None, wpid=None, protocol=None, registers=None,
 # Mice
 
 _D('Wireless Mouse M175', protocol=2.0, wpid='4008')
-_D('Wireless Mouse M185')
+_D('Wireless Mouse M185 new', codename='M185n', protocol=4.5, wpid='4054',
+				settings=[
+							_FS.lowres_smooth_scroll(),
+				])
+_D('Wireless Mouse M185 old', codename='M185o', protocol=4.5, wpid='4055',
+				settings=[
+							_FS.lowres_smooth_scroll(),
+				])
```

Comment:
Auto-detection requires the device to be on, could that have been the problem?
The WPID is different, that should be good enough to differentiate devices?